### ISSUE TITLE
Merriweather Sans doesn't contains cyrillic

### ISSUE THREAD

**Issue by alexeyten** [google/fonts#2825]
https://fonts.google.com/specimen/Merriweather+Sans?subset=cyrillic-ext&preview.text=Нет кириллицы No cyrillic&preview.text_type=custom

![image](https://user-images.githubusercontent.com/44377/99775178-e7ed4800-2b1f-11eb-9405-7bd70487c6ff.png)

**Comment by thlinard** [comment]
Indeed! Something wrong in https://github.com/google/fonts/blob/master/ofl/merriweathersans/METADATA.pb

> ```
> subsets: "cyrillic-ext"
> subsets: "latin"
> subsets: "latin-ext"
> subsets: "menu"
> subsets: "vietnamese"
> ```

`subsets: "cyrillic-ext"` should be removed.

@EbenSorkin Merriweather Sans is advertised as supporting many Cyrillic languages, but I can't find any Cyrillic characters in the fonts?

**Comment by alexeyten** [comment]
Doesn't show `ru` or `kz` or any other cyrillic-based language.
```
$ fc-scan --format="%{lang}\n" MerriweatherSans-Regular.ttf  
aa|af|ast|ay|az-az|bi|bin|br|bs|ca|ch|co|cs|cy|da|de|en|eo|es|et|eu|fi|fj|fo|fr|fur|fy|ga|gd|gl|gn|gv
|haw|ho|hr|hu|ia|ig|id|ie|io|is|it|ki|kl|kw|la|lb|lt|lv|mg|mh|mt|nb|nds|nl|nn|no|nr|nso|ny|oc|om|pl|pt
|rm|ro|se|sk|sl|sm|sma|smj|smn|so|sq|ss|st|sv|sw|tk|tl|tn|to|tr|ts|uz|vi|vo|vot|wa|wen|wo|xh
|yap|zu|an|crh|csb|fil|hsb|ht|jv|kj|ku-tr|kwm|lg|li|ms|na|ng|nv|pap-an|pap-aw|qu|quz|rn|rw|sc|sg|sn|su|ty|za
```
